Services/Component/classes/class.ilObjComponentSettingsGUI.php

```diff
@@ -517,17 +502,28 @@ function updatePlugin()
 		$pl = ilPlugin::getPluginObject($_GET["ctype"], $_GET["cname"],
 			$_GET["slot_id"], $_GET["pname"]);
 
-		$result = $pl->update();
-		
+		$this->update($pl);
+	}
+
+	/**
+	 * Execute update and ctrl reload
+	 *
+	 * @param ilPlugin 	$plugin
+	 *
+	 * @return null
```

Comment:
`null` is technically correct:

```php
function foo() {
}

assert(foo() === null);
```

I just looked that stuff up in [PHPDocumentor|https://www.phpdoc.org/docs/latest/references/phpdoc/types.html] and I'm not sure I really see the reason why `void` exist at all, besides (maybe) tradition. Does PHPStorm insert `void` automatically if there is no return in the method?